packages/vscode-apollo/src/testRunner/jest-config.ts

```diff
@@ -0,0 +1,20 @@
+import { resolve } from "path";
+
+export const config = {
+  // This seems to have no effect, though the need may arise when running more tests
```

Comment:
What does `this` refer to here? Can we outline this file in a larger comment explaining the what and the why?

Reply:
`This` refers to `runInBand` - it was here via prior art.
...it doesn't even exist in the docs so I'm just going to remove it.

To your second point, this is just a jest config - I can add an @see that points to the documentation. Additionally, I can add some comments about the less common stuff - `testEnvironment` and `setupTestFrameworkScriptFile` if you think that's useful?